ivy/functional/backends/paddle/statistical.py

```diff
@@ -1,7 +1,10 @@
 # global
 
 torch_scatter = None
-from typing import Union, Optional, Sequence
+
+from typing import Union, Optional, Sequence, Tuple
+
+import paddle
```

Comment:
This line is causing a listing error too.

Reply:
Yes,it was duplicate import and it is fixed now. I made a new pull request https://github.com/unifyai/ivy/pull/15544

Reply:
Can we resolve it here instead?

Reply:
I have resolved it and already commited here .